rollup.config.js

```diff
@@ -18,42 +18,49 @@ const tsOpts = {
 };
 
 const globals = {
-    react: 'automat.react',
-    emotion: 'automat.emotion', // TODO remove this dependency
-    '@emotion/core': 'automat.emotionCore',
-    'emotion-theming': 'automat.emotionTheming',
+    react: 'guardian.automat.react',
+    emotion: 'guardian.automat.emotion', // TODO remove this dependency
+    '@emotion/core': 'guardian.automat.emotionCore',
+    'emotion-theming': 'guardian.automat.emotionTheming',
 };
 
-export default {
-    input: 'src/components/modules/ContributionsEpic.tsx',
-    output: {
-        file: 'dist/modules/Epic.js',
-        format: 'es',
-        sourcemap: process.env.NODE_ENV === 'production' ? false : 'inline',
-    },
-    external: id => Object.keys(globals).some(key => id.startsWith(key)),
-    plugins: [
-        resolveNode(),
-        commonjs(),
-        json(),
-        babel({
-            extensions: ['.ts', '.tsx', '.js', '.jsx', '.es6', '.es', '.mjs'],
-            plugins: [['emotion', { sourceMap: false }]],
-            presets: [
-                [
-                    '@babel/preset-env',
-                    {
-                        targets: {
-                            ie: '11',
+const config = [
+    ['src/components/modules/ContributionsEpic.tsx', 'dist/modules/Epic.js'],
+    ['src/components/modules/Banner.tsx', 'dist/modules/Banner.js'],
+].map(([entryPoint, name]) => {
```

Comment:
I also have a version that built 2 chunks rather than 2 bundles (I think that's the right terminology). Check the commits, but I think this is the right way since there isn't a link between Epic and Banner modules.

Reply:
This sounds legit to me, but I'll defer to @nicl who's got a better understanding of the rollup config!

Reply:
Agree, they should be completely separate bundles.